Sources/SwiftHtml/Attributes/Events.swift

```diff
@@ -96,6 +96,200 @@ public extension Attribute {
     }
 }
 
+public extension Attribute {
```

Comment:
Please ignore this. It was a mistake. Language is a global attribute. It has been moved in a subsequent commit